sdk/storage/azure-storage-blob/src/main/java/module-info.java

```diff
@@ -6,6 +6,7 @@
 
     requires com.azure.storage.internal.avro;
     requires com.azure.core;
+    requires java.sql;
```

Comment:
Why did this line get added? I did not see anything related to SQL as part of this update.